strum_macros/src/helpers/metadata.rs

```diff
@@ -14,6 +15,7 @@ pub mod kw {
 
     // enum metadata
     custom_keyword!(serialize_all);
+    custom_keyword!(crate_path);
```

Comment:
Did you say serde uses "crate?" Can we also do that for consistency?

Reply:
`crate` is a Rust keyword, is it okay if I use `Crate` instead?

Reply:
sorry for the delay getting back to you. thanks for your patience. I would prefer crate if we can since that's what serde does. Is there a technical reason we can't do that?

Reply:
Just overcome the technical difficulties, see the latest commit, now we have

```rust
#[derive(Debug, Eq, PartialEq, EnumIter)]
#[strum(crate = "nested::module::strum")]
enum Week {
    Sunday,
    Monday,
    Tuesday,
    Wednesday,
    Thursday,
    Friday,
    Saturday,
}
```